src/elements/table/table-wrapper/table-wrapper.visual.spec.ts

```diff
@@ -0,0 +1,75 @@
+import { html } from 'lit';
+import { classMap } from 'lit/directives/class-map.js';
+
+import {
+  describeEach,
+  describeViewports,
+  visualRegressionFixture,
+  visualDiffDefault,
+} from '../../core/testing/private.js';
+
+import './table-wrapper.js';
+
+describe(`sbb-table-wrapper`, () => {
+  let root: HTMLElement;
+
+  const cases = {
+    size: ['m', 's'],
+    negative: [false, true],
+    striped: [false, true],
+    // TODO add scrollbar cases
+  };
+
+  describeViewports({ viewports: ['zero', 'medium'] }, () => {
+    describeEach(cases, ({ size, negative, striped }) => {
```

Comment:
In my eyes, the tests of the size could be separate to negative/striped. the size is not influenced by negative nor striped. To save snapshots.